frontend/src/APIClients/BaseAPIClient.ts

```diff
@@ -33,6 +33,7 @@ baseAPIClient.interceptors.request.use(async (config: AxiosRequestConfig) => {
     authHeaderParts.length >= 2 &&
     authHeaderParts[0].toLowerCase() === "bearer"
   ) {
+    // TODO: define expected JWT return type
     const decodedToken: any = jwt.decode(authHeaderParts[1]);
```

Comment:
@sherryhli @chenrichard10  @zm2he 
Anyone know what is supposed to be the format of the object returned? Changing this from any to the expected object.

Reply:
I'll look into it!

Reply:
Should be `string | { [key: string]: any; } | null` but typing it as such would result in an error when accessing the `exp` property below. Any way to work around this?

While you're here could you add a line to skip the `exp` check if `decodedToken` is a string?